catalog/spatial/registry/pom.xml

```diff
@@ -157,6 +157,6 @@
         <module>registry-federation-admin-impl</module>
         <module>registry-app</module>
         <module>registry-admin-modules</module>
-
+        <module>registry-report-viewer</module>
```

Comment:
Since this isn't the actual viewer we might want to name it something like registry-report-action-provider